src/indexing/BaseEventIndexManager.ts

```diff
@@ -18,14 +18,14 @@ export interface MatrixEvent {
     type: string;
     sender: string;
     content: {};
-    event_id: string;
-    origin_server_ts: number;
+    eventId: string;
+    originServerTs: number;
```

Comment:
Not sure these are ok to rename either

Reply:
Very unsafe. They must remain underscored.